sys/include/net/gnrc/pktdump.h

```diff
@@ -28,13 +28,23 @@
 extern "C" {
 #endif
 
+/**
+ * @defgroup net_gnrc_pktdump_conf GNRC PKTDUMP compile configurations
+ * @ingroup net_gnrc_conf
+ * @{
+ */
 /**
  * @brief   Message queue size for the pktdump thread
  */
-#ifndef GNRC_PKTDUMP_MSG_QUEUE_SIZE
-#define GNRC_PKTDUMP_MSG_QUEUE_SIZE     (8U)
+#ifdef CONFIG_GNRC_PKTDUMP_MSG_QUEUE_SIZE_EXP
+#define CONFIG_GNRC_PKTDUMP_MSG_QUEUE_SIZE  (1<<CONFIG_GNRC_PKTDUMP_MSG_QUEUE_SIZE_EXP)
```

Comment:
Does this need to be renamed to `CONFIG_` if it is not exposed to Kconfig?

Reply:
Well, if this macro is not configurable then the prefix is not needed. But also, the `ifndef` can be removed as the user would configure the queue size via `CONFIG_GNRC_PKTDUMP_MSG_QUEUE_SIZE_EXP`

Reply:
Fine. I shall update the header based on PR #14086